projects/1.18/assets/electrodynamics/electrodynamics/lang/zh_cn.json

```
@@ -1 +1,555 @@
+  "block.electrodynamics.resourceblockaluminum" : "铝块",
+  "block.electrodynamics.resourceblockchromium" : "铬块",
+  "block.electrodynamics.resourceblockstainlesssteel" : "不锈钢块",
+  "block.electrodynamics.resourceblockvanadiumsteel" : "钒钢合金块",
+  "block.electrodynamics.resourceblockhslasteel" : "高强度低合金钢块",
+  "block.electrodynamics.resourceblocktitanium" : "钛块",
+  "block.electrodynamics.resourceblocktitaniumcarbide" : "碳化钛块",
+  "block.electrodynamics.orelead" : "方铅矿石",
+  "block.electrodynamics.oremonazite" : "独居石矿石",
+  "block.electrodynamics.oresilver" : "辉银矿石",
+  "block.electrodynamics.oresulfur" : "硫矿石",
+  "block.electrodynamics.oreniter" : "硝石矿石",
+  "block.electrodynamics.orethorianite" : "方钍石矿石",
+  "block.electrodynamics.oretin" : "锡石矿石",
+  "block.electrodynamics.oreuraninite" : "铀矿石",
+  "block.electrodynamics.orevanadinite" : "钒铅矿石",
+  "block.electrodynamics.orelepidolite" : "锂云母矿石",
+  "block.electrodynamics.orealuminum" : "铝土矿石",
+  "block.electrodynamics.orechromite" : "铬铁矿石",
+  "block.electrodynamics.orerutile" : "金红石矿石",
+  "block.electrodynamics.orehalite" : "石盐矿石",
+  "block.electrodynamics.oremolybdenum" : "钼矿石",
+  "block.electrodynamics.orefluorite" : "氟石矿石",
+  "block.electrodynamics.oresylvite" : "钾石盐矿石",
+  "block.electrodynamics.deepslateorelead" : "深层方铅矿石",
+  "block.electrodynamics.deepslateoremonazite" : "深层独居石矿石",
+  "block.electrodynamics.deepslateoresilver" : "深层辉银矿石",
+  "block.electrodynamics.deepslateoresulfur" : "深层硫矿石",
+  "block.electrodynamics.deepslateoreniter" : "深层硝石矿石",
+  "block.electrodynamics.deepslateorethorianite" : "深层方钍矿石",
+  "block.electrodynamics.deepslateoretin" : "深层锡石矿石",
+  "block.electrodynamics.deepslateoreuraninite" : "深层铀矿石",
+  "block.electrodynamics.deepslateorevanadinite" : "深层钒铅矿石",
+  "block.electrodynamics.deepslateorelepidolite" : "深层锂云母矿石",
+  "block.electrodynamics.deepslateorealuminum" : "深层铝土矿石",
+  "block.electrodynamics.deepslateorechromite" : "深层铬铁矿石",
+  "block.electrodynamics.deepslateorerutile" : "深层金红石矿石",
+  "block.electrodynamics.deepslateorehalite" : "深层石盐矿石",
+  "block.electrodynamics.deepslateoremolybdenum" : "深层钼矿石",
+  "block.electrodynamics.deepslateorefluorite" : "深层氟石矿石",
+  "block.electrodynamics.deepslateoresylvite" : "深层钾石盐矿石",
+  "block.electrodynamics.raworeblocktin" : "粗锡块",
+  "block.electrodynamics.raworeblocksilver" : "粗银块",
+  "block.electrodynamics.raworeblocklead" : "粗铅块",
+  "block.electrodynamics.concreteregular" : "加固混凝土",
+  "block.electrodynamics.concretetile" : "加固混凝土瓦块",
+  "block.electrodynamics.concretebricks" : "加固混凝土砖",
+  "block.electrodynamics.coalgenerator" : "燃煤发电机",
+  "block.electrodynamics.coalgeneratorrunning" : "燃煤发电机（运行中）",
+  "block.electrodynamics.solarpanel" : "太阳能板",
+  "block.electrodynamics.advancedsolarpanel" : "高级太阳能板",
+  "block.electrodynamics.electricfurnace" : "电炉",
+  "block.electrodynamics.electricfurnacerunning" : "电炉（运行中）",
+  "block.electrodynamics.electricfurnacedouble" : "双联电炉",
```

```suggestion
  "block.electrodynamics.electricfurnacedouble" : "冶炼厂",
```
我先说我的理由
首先.1.16.5的译者采用了电炉-冶炼厂-高级冶炼厂的翻译
我认为主要是受container词条的影响
  "container.electricfurnace0" : "Electric Furnace",
  "container.electricfurnace1" : "Smelting Factory",
  "container.electricfurnace2" : "Advanced Smelting Factory",
此外，实际的机器在升级后速度并未明显提升，只是可以同时运行2/3个烧炼进程，因此n联电炉似乎也可以
不过我还是支持作者原文的工厂
  "container.electricarcfurnace0" : "Electric Arc Furnace",
  "container.electricarcfurnace1" : "Blasting Factory",
  "container.electricarcfurnace2" : "Advanced Blasting Factory",
在1.18.1新更新的电弧炉词条中也是和电炉一样采用了电弧炉-爆破厂-高级爆破厂的称呼

---

我觉得尊重block name没有毛病，上面也讨论过一波了

---

建议n联